modules/relaidoBidAdapter.js

```diff
@@ -349,6 +352,44 @@ function getBannerSizes(bidRequest) {
   return parseSizesInput(sizes).join(',');
 }
 
+function getTargeting(bidRequest) {
```

Comment:
I think the pre-auction module does most of this work

Reply:
@patmmccann 
I have some code that is a bit similar, but this is not what I want.
https://github.com/prebid/Prebid.js/blob/master/src/secureCreatives.js#L164-L171